src/Application/Migrations/15_function_ticket_state.sql

```diff
@@ -2,7 +2,7 @@ BEGIN;
 
 SET ROLE TO postgres;
 
-CREATE OR REPLACE FUNCTION ticket_state_next(param_project_id bigint, param_ticket_type enum_ticket_type, param_ticket_state enum_ticket_state)
+CREATE OR REPLACE FUNCTION ticket_state_next(param_project_id bigint, param_ticket_type enum_ticket_type, param_ticket_state enum_ticket_state, param_ticket_id bigint default NULL)
```

Comment:
Mh, not really convinced having that extra parameter is a good idea.
Before this change, the function was generic, independent of a specific ticket, hence all the parameters to answer the question at hand. But now it's for a single ticket, so you could go ahead and just have ticket_id as only parameter, since all the other information is stored in the database as well.

Reply:
> Mh, not really convinced having that extra parameter is a good idea.

What do you suggest? Moving to `ticket_id` as only parameter or `encoding_profile_[version_]_id`?

Reply:
Both would work. I don't have a strong opinion on this.
Are there any callers, where no ticket_id is available or hard to determine?
I'm all for simplifying, if possible.

Reply:
This should behave the same way as the old function when calling, without `ticket_id`, but we should check that again.